dfndb/views.py

```diff
@@ -60,7 +60,7 @@ def post(self, request, *args, **kwargs):
         except FormulaError as e:
             messages.error(
                 request,
-                f"{e} - Formula is not valid. Please check your formula or enter the mass manually.",
+                f"{e} - Formula is not valid. Please check your formula.",
```

Comment:
Not to be addressed in this PR, but I don't like that this change needed to be made twice. Perhaps there should be some unification between the "Create" and "Update" views.

Reply:
Yes the challenge is that the `post()` method needs to be overridden for each view in a slightly different way (depending on whether the object exists already). There's probably a way to do this with some if statements.